notify/notify_test.go

```diff
@@ -91,26 +91,35 @@ func TestDedupStageNeedsUpdate(t *testing.T) {
 		firingAlerts   map[uint64]struct{}
 		resolvedAlerts map[uint64]struct{}
 		repeat         time.Duration
+		resolve        bool
 
-		res    bool
-		resErr bool
+		res bool
 	}{
 		{
+			// No matching nflog entry should update.
 			entry:        nil,
 			firingAlerts: alertHashSet(2, 3, 4),
 			res:          true,
 		}, {
+			// No matching nflog entry shouldn't update if no alert fires.
+			entry:          nil,
+			resolvedAlerts: alertHashSet(2, 3, 4),
+			res:            false,
+		}, {
+			// Different sets of firing alarms should update.
```

Comment:
Could you update `alarms` in this comment and all the others to `alerts`? Super nit-picky, sorry !

Reply:
No problem. Not sure why I wrote "alarms" here...